articles/mysql/single-server-whats-new.md

```diff
@@ -48,6 +48,21 @@ This release of Azure Database for MySQL - Single Server includes the following
 
   This release provides General Availability of planned maintenance notifications in Azure Database for MySQL - Single Server. For more information, see the article [Planned maintenance notification](concepts-planned-maintenance-notification.md).
 
+- **Enabled the parameter redirect_enabled by default**
+
+   With this release, the redirect_enabled parameter is enabled by default. Redirection aims to reduce network latency between client applications and MySQL servers by allowing applications to connect directly to backend server nodes. Support for redirection in PHP applications is available through the [mysqlnd_azure](https://github.com/microsoft/mysqlnd_azure) extension, developed by Microsoft. For more information, see [Connect to Azure Database for MySQL with redirection](https://docs.microsoft.com/en-us/azure/mysql/howto-redirection).
+             
+   Note: 
+   
+   . Enabling redirection is not supported with private link setup , if it is already enabled , you will have connections issue and to resolve it make sure to disable the redirectiom  by setting the redirect_enabled parameter to “OFF”, and restart the PHP application. 
```

Comment:
Private link setup does not work with re-direction. If you are using Private link for Azure Database for MySQL, you might face connection issue when redirection drivers are used. To resolve the issue make sure the setting redirect_enabled parameter to “OFF”, and restart the client application.